### DISCUSSION TITLE
[vite-plugin-sveltekit-compile] Cannot read properties of undefined (reading 'file')

### DISCUSSION THREAD

**Discussion by iunknown80** [open-webui/open-webui#5044] [General]
# Bug Report

## Installation Method

I am using **Install from Open WebUI Github Repo** method in Windows.

## Environment

- **Open WebUI Version:** [v0.3.16]
- **Operating System:** [Windows Server 2019]

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [x] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Description

 I got the latest code from Git. When running the _npm run build_ command,

`git clone https://github.com/open-webui/open-webui.git
cd open-webui

copy .env.example .env

npm install
npm run build`

 I am getting the following error. 

## Logs and Screenshots

✓ 895 modules transformed.
x Build failed in 45.18s
error during build:
**[vite-plugin-sveltekit-compile] Cannot read properties of undefined (reading 'file')**
    at file:///H:/source/repos/open-webui/node_modules/@sveltejs/kit/src/exports/vite/build/build_server.js:61:83
    at Array.forEach (<anonymous>)
    at build_server_nodes (file:///H:/source/repos/open-webui/node_modules/@sveltejs/kit/src/exports/vite/build/build_server.js:33:22)
    at Object.handler (file:///H:/source/repos/open-webui/node_modules/@sveltejs/kit/src/exports/vite/index.js:760:5)
    at file:///H:/source/repos/open-webui/node_modules/rollup/dist/es/shared/node-entry.js:19892:40
    at async PluginDriver.hookParallel (file:///H:/source/repos/open-webui/node_modules/rollup/dist/es/shared/node-entry.js:19814:17)
    at async file:///H:/source/repos/open-webui/node_modules/rollup/dist/es/shared/node-entry.js:20817:13
    at async catchUnfinishedHookActions (file:///H:/source/repos/open-webui/node_modules/rollup/dist/es/shared/node-entry.js:20237:16)
    at async build (file:///H:/source/repos/open-webui/node_modules/vite/dist/node/chunks/dep-BzOvws4Y.js:65337:16)
    at async CAC.<anonymous> (file:///H:/source/repos/open-webui/node_modules/vite/dist/node/cli.js:828:5)
## Additional Information

I even tried to install vite using "npm i vite". Still no luck. All of these I am doing directly from the command line following the instruction from OpenWebUI.

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by iunknown80** [comment]
Any help on this would be appreciated.

**Comment by sudocurse** [comment]
what's your node/npm version?

**Comment by jeremykei** [comment]
Saw the drive was H. Assuming its the same issue I met, try executing "npm run build" on the actual drive H: instead of say, from C drive. 

Running from c: as mapped folder
![image](https://github.com/user-attachments/assets/47d31de0-14dd-4084-818a-418147ae6a34)

Running from d: as actual physical location
![image](https://github.com/user-attachments/assets/b0a3d099-6d30-4c43-9137-f7334af7404b)
